assets/wizards/popups/utils.js

```diff
@@ -47,9 +45,15 @@ export const isCustomPlacement = popup => {
 export const isInline = prompt => ! isOverlay( prompt );
 
 const placementMap = {
-	center: __( 'Center Overlay', 'newspack' ),
 	top: __( 'Top Overlay', 'newspack' ),
+	'top-left': __( 'Top Left Overlay', 'newspack' ),
```

Comment:
Minor nitpick that applies to all of these new properties:

```suggestion
	'top_left': __( 'Top Left Overlay', 'newspack' ),
```

For JS properties, it's best to use an `_` or even just camelcase as you can't use dot notation (`objectName.property_name`) if the property name contains a hyphen. Since we already have `above_header` I think we can follow that format for all multi-word properties.